platform/ios/app/MBXViewController.m

```diff
@@ -2202,6 +2202,23 @@ - (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style
     // that a device with an English-language locale is already effectively
     // using locale-based country labels.
     _localizingLabels = [[self bestLanguageForUser] isEqualToString:@"en"];
+
```

Comment:
I don't understand why adding behavior under `didFinishLoadingStyle`? Could it be a new runtimeStyling case?

Reply:
This is just for debugging purposes.